apps/mobile/src/components/badges/network-badge.tsx

```diff
@@ -0,0 +1,25 @@
+import { AppRoutes } from '@/routes';
```

Comment:
I'm not sure about having state connected components under `src/components` 

I would think they should be a feature?

Reply:
Yeah, we have some dep cruiser rules for this in the extension we need to add to mobile...

<img width="805" alt="Screenshot 2024-11-15 at 1 36 49 PM" src="https://github.com/user-attachments/assets/b76ec6b8-2ab7-4942-910e-22a104e1c73a">

Reply:
My understanding is similar but that if a shared component is connected to state, it then becomes a `feature` rather than a `component`. That's just how I have seen things done in the past generally. 

I'm not sure if we can import one feature to another with those cruiser rules? Maybe they are enforcing a different setup we prefer. 

Usually only dumb, non state connected components would go to `src/components/` then state connected ones into `src/features`. If a component is used in multiple features:
- it could be made more dumb and receive props rather than hook into state
- if it needs to fetch state be promoted to a feature